services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/helpers/PathHelper.groovy

```diff
@@ -60,7 +60,7 @@ class PathHelper {
             def involvedIsl = topology.isls.find {
                 (it.srcSwitch.dpId == src.switchId && it.srcPort == src.portNo && it.dstPort == dst.portNo &&
```

Comment:
Should probably use safe navigation here too

Reply:
Ok, let's add it here too.